js/src/product-feed/issues-table-card/issues-solved.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { capitalize } from 'lodash/string';
+import { createInterpolateElement } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+import { __experimentalText as Text, Dashicon } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { ISSUE_TYPE_PRODUCT } from '.~/constants';
+import useActiveIssueType from '.~/hooks/useActiveIssueType';
+
+/**
+ * This component renders a message when no issues of the
+ * active issue type are pending to solve.
+ *
+ * @return {JSX.Element} The component
+ */
+const IssuesSolved = () => {
+	const issueType = useActiveIssueType();
+	const otherIssueType =
+		issueType === ISSUE_TYPE_PRODUCT ? 'account' : 'product';
+
+	const map = {
+		otherIssueTab: <strong>{ capitalize( otherIssueType ) } Issues</strong>,
+		otherIssueType: <span>{ otherIssueType }</span>,
+	};
+
+	return (
+		<div className="gla-issues-solved">
+			<Dashicon icon="yes-alt" className="gla-issues-solved__icon" />
+			<Text variant="subtitle">
+				{ sprintf(
+					// translators: %s: Active issue type (account or product)
+					__( 'All %s issues resolved', 'google-listings-and-ads' ),
+					issueType
+				) }
```

Comment:
❓  Same as above. It should avoid composing i18n string programmatically if it's possible.

Reply:
Updated here 

https://github.com/woocommerce/google-listings-and-ads/pull/1196/commits/f72fa4f23758c31147499fe1a2607a7b805224c5

❓ Why is not allowed/recommended to use the placeholder?

Reply:
I guess this is not a strict rule, just that for translation purposes. The dynamic i18n strings generated programmatically can not be translated completely. For example, "All %s issues resolved", English users will see the correct "All account issues resolved", but Japanese language users will see "すべての **account** の問題が解決しました" rather than "すべての**アカウント**の問題が解決しました".

In addition, in some languages such as Korean, spaces are part of the grammar and may be used to break up words/sentences or as auxiliary verbs. The same paragraph may have a completely different meaning depending on the spaces. There may be insufficient context, or the dynamic content itself may contain spaces, causing the final composed result of the translated text and dynamic content to be incorrect.

On the other hand, for example, the string "However, there are issues affecting your **\<otherIssueType /\>** that needs to be resolved." might not be semantically insufficient for translators. They have no way of knowing what words are in the **\<otherIssueType /\>**, which may result in a lack of context, making the translation potentially incorrect.

Therefore, avoiding dynamic generation of i18n strings whenever possible is often a priority.

Reply:
Gotcha! Thanks for the detailed explanation. Didn't think about the order of the words and stuff like that.